concreate/tools.py

```diff
@@ -65,7 +65,7 @@ def _generate_url(self):
                             .replace('#hash#', self.sums[alg]))
                 break
 
-    def verify(self):
+    def _verify(self):
```

Comment:
Even if called during fetch() I would still keep this method "public".

Reply:
OK.